### PR TITLE
[FAB-17574] Support adding an application org to a channel config

### PR DESCRIPTION
**Pull request by wuxuer**
Description

Modify an existing channel configuration to add an application org.

Related issues

https://jira.hyperledger.org/browse/FAB-17574

Signed-off-by: xu wu <wuxu1103@163.com>

#### Type of change

- New feature

### CODE REVIEW

File: pkg/config/application_test.go [hunk 1]
```diff
@@ -10,10 +10,12 @@ import (
 	"bytes"
 	"testing"
 
+	"github.com/hyperledger/fabric/common/tools/protolator"
+	"github.com/hyperledger/fabric/common/tools/protolator/protoext/peerext"
+
```
Comment (DereckLuo) [inline pkg/config/application_test.go:15]: nit: empty line in import.

**Review by DereckLuo — COMMENTED** [review]
LGTM

File: pkg/config/application.go [hunk 2]
```diff
@@ -209,3 +209,18 @@ func getApplicationOrg(config *cb.Config, orgName string) (*cb.ConfigGroup, erro
 	}
 	return org, nil
 }
+
+// AddApplicationOrg adds an organization to an existing config's Application configuration.
```
Comment (sykesm) [inline pkg/config/application.go:213]:
```suggestion
// AddApplicationOrg adds an organization to an existing Application configuration.
```

We really need to move away from possessives in the doc. I realize this is not the only place we’re doing it but I wanted to highlight it.

File: pkg/config/application_test.go (hunk 1 above)
Comment (sykesm) [reply pkg/config/application_test.go:15]: agreed; we don’t need to separate these imports into a group.

**Review by sykesm — APPROVED** [review]

**Pull request merged**